src/domain/participant/index.js

```diff
@@ -59,8 +60,10 @@ const { destroyParticipantEndpointByParticipantId } = require('../../models/part
 
 const create = async (payload) => {
   try {
+    logger.debug('creating participant with payload', payload)
     return ParticipantModel.create({ name: payload.name, isProxy: !!payload.isProxy })
   } catch (err) {
+    logger.error('error creating participant', { err, payload })
```

Comment:
I'm not sure it will work, I assume we'll have in logs something like this:
```
2024-08-21T09:41:51.659Z - error: - error creating participant: - { "err": {} , payload: { // some data } }
```
for now in logger we can log error details only if it passed like this: logger.error('error message', err)

Si in this particular case, if we want to have `payload` be logged in every logs withing the function, we can use logger.child() approach:
```
   const log = logger.child(payload)  // or const log = logger.child({ payload }) to have payload-field 
   ...
   logger.debug('creating participant with payload')
   ...
   logger.error('error creating participant', err)
```

Reply:
done, here and throughout.